tests/testthat/test-build-citation-authors.R

```diff
@@ -34,3 +34,12 @@ test_that("source link is added to citation page", {
   lines <- read_lines(path(path, "docs", "authors.html"))
   expect_true(any(grepl("<code>inst/CITATION</code></a></small>", lines)))
 })
+
+test_that("multiple citations all have HTML and BibTeX formats", {
+  path <- test_path("assets/site-citation/multi")
+  citations <- data_citations(path)
+  expect_length(citations, 2)
+  expect_true(all(lengths(citations) == 2))
```

Comment:
This would be better as `expect_equal(length(citations), c(2, 2))` because you'll get a much more informative failure message.  That would also let you remove the separate `expect_length(citations, 2)` assertion.